tapdb/sqlc/migrations/000028_asset_meta_decimal_display.up.sql

```diff
@@ -0,0 +1,5 @@
+-- We add a set of new minting/universe related columns. Those will all be
+-- added to the meta record as odd (optional) TLV fields. Therefore, they will
+-- not be set on older assets, which is why we need to make them nullable here.
+ALTER TABLE assets_meta ADD COLUMN meta_decimal_display INTEGER;
+ALTER TABLE assets_meta ADD COLUMN meta_canonical_universe VARCHAR;
```

Comment:
`meta_canonical_universe` entries will be size limited on TLV record p2p limit, right?

Reply:
Good catch. It was not. Also added length validation in the `MetaReveal.Validate()` method.